src/NLPModels.jl

```diff
@@ -344,7 +344,7 @@ conscale(::AbstractNLPModel, ::AbstractVector) =
   throw(NotImplementedError("conscale"))
 
 if Pkg.installed("MathProgBase") != nothing
-  include("mpb_model.jl")
```

Comment:
What happened to `mpb_model`?

Reply:
It's being `include`d by `nlsmpb_model.jl`. It uses many MPB things in common.

Reply:
I missed that. Thanks!